### ISSUE TITLE
[Bug]: FC0005 Flase positive for table field names

### ISSUE THREAD

**Issue by fridrichovsky** [ALCops/Analyzers#558]
### Analyzer

FormattingCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

FC0005

### Description

Tables that use bad field names generate false positive diagnostic.

### AL code to reproduce

```al
        if not IsConfigured then begin
            NewNotification.Id := this.GetResMgtSetupId();
            NewNotification.Message := SetupLbl;
            NewNotification.Send();
        end;
```
Same is for field names in qoutes.

```al
        if not IsConfigured then begin
            NewNotification."Id" := this.GetResMgtSetupId();
            NewNotification."Message" := SetupLbl;
            NewNotification.Send();
        end;
```

**Comment by Arthurvdv** [comment]
I'm a little confused about `NewNotification` here. Is that a variable of the [Notification data type](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/notification/notification-data-type)?

<img width="768" height="234" alt="Image" src="https://github.com/user-attachments/assets/dba435a5-cbe3-49b2-bf09-e306065d3f14" />

If so, I’m not sure this would be a false positive. Could you clarify what I might be missing?

**Comment by fridrichovsky** [comment]
Hello I am sorry my mistake we have same table name in module and it confused me. You are right and it works correctly.
